src/components/dialog/dialog.stories.tsx

```diff
@@ -85,7 +85,7 @@ export const Editable: StoryFn = () => {
             </Button>
           }
         >
-          <Typography variant="h3" mb="32px">
+          <Typography variant="h2" mb="32px">
```

Comment:
**question:** Is this just a semantics change as the use of `h3` was incorrect?

Reply:
That's right, its to address a small AXE violation